### DISCUSSION TITLE
sx.fukt.lol removed for bad uptime.

### DISCUSSION THREAD

**Discussion by iotalol** [searxng/searx-instances#284] [General]
Hi, my instance was removed for bad uptime and I would like to see if I can get re-listed. My colo where I was running SearXNG had power issues and I resolved that by adding a UPS, the current uptime on my SearXNG instance is 14 days with regular planned 2 week reboots.

**Comment by unixfox** [comment]
Please open a new issue for that: https://github.com/searxng/searx-instances/issues
